_docs/aop.md

```diff
@@ -248,15 +258,15 @@ class AnExampleAspect
 
 Aspect classes have to be implemented within your webapp, namely within the `META-INF`, `WEB-INF` or `common` directory.
 
-> Please note that aspects MUST NOT have a constructor requiring parameters.
+> Please note that aspects CANNOT contain a constructor requiring parameters.
```

Comment:
Ok. How is this?

Please note that the constructor of aspect classes MUST NOT require parameters.